res/css/photon/message-bar.css

```diff
@@ -5,53 +5,66 @@
 /* See https://design.firefox.com/photon/components/message-bars.html */
 .photon-message-bar {
   display: flex;
-  width: 100%;
   min-height: 32px;
   box-sizing: border-box;
   align-items: center;
-  padding: 0 4px;
```

Comment:
This padding was useful when we weren't having an icon at the left, but now we always have one.